packages/eslint-plugin/src/rules/no-exported-css/__tests__/rule.test.ts

```diff
@@ -685,6 +685,39 @@ const createTestCases = (importSource: string) =>
         `,
         errors: [],
       },
+      {
+        code: `
+          import { css, styled } from '${importSource}';
+
+          const styles = ${createStyle(level)};
+
+          export const Component = styled.div\`\${styles}\`;
+        `,
+        errors: [],
+      },
+      {
+        code: `
+          import { css, styled as styled2 } from '${importSource}';
+
+          const styles = ${createStyle(level)};
+
+          export const Component = styled2.div\`\${styles}\`;
+        `,
+        errors: [],
+      },
+      {
+        code: `
+          import { css, styled } from '${importSource}';
+
+          const buttonStyle = css\`
+            display: flex;
+            align-items: center;
+            justify-content: center;\`;
+
+          export const Component = styled.button(buttonStyle);
+        `,
+        errors: [],
+      },
```

Comment:
I've seen something like below in Confluence.

```
export const Foo = styled.div`
    color: black;
    ${({ isXEnabled }) => isXEnabled ? css`width: 100px` : undefined
`
```

Can you please add a test case for it?

Reply:
- [ ] add test